src/pkg/runner/actions.go

```diff
@@ -95,7 +95,7 @@ func getUniqueTaskActions(actions []types.Action) []types.Action {
 	return uniqueArray
 }
 
-func RunAction[T any](action *types.BaseAction[T], envFilePath string, variableConfig *variables.VariableConfig[T]) error {
+func runAction[T any](action *types.BaseAction[T], envFilePath string, variableConfig *variables.VariableConfig[T]) error {
```

Comment:
Sorry wasn't clear earlier - this should have a comment - it would be an entrypoint for library users (like Zarf) in the future.